astropy/io/ascii/daophot.py

```diff
@@ -181,7 +179,7 @@ def get_cols(self, lines):
                 col.format = fmt
 
         # Set column start and end positions.
-        col_width = sum(self.col_widths, [])
+        col_width = list(itt.chain.from_iterable(self.col_widths))
```

Comment:
```suggestion
        col_width = list(itt.chain(*self.col_widths))
```
Don't know, did @mhvk's suggestion apply here as well?

Reply:
It can. This was just a 1-liner either way and `chain.from_iterable` is faster than successive packing / unpacking.

Reply:
This bothered me less than the other one, maybe because the other had just `v`, so the method really distracted from the variable.

My sense would be that this is fine and not worth running CI for again!